### DISCUSSION TITLE
I want to know how you make this project using the Terraform and Terragrunt

### DISCUSSION THREAD

**Discussion by saxenachaitanya** [particuleio/teks#107] [General]
Hello 
I have seen your project teks It is awesome to run the production grade cluster on EKS , I also want to learn how you made this possible can u please suggest me how can I get into this how can I learn the things soo that I can also able to make the EKS cluster like u guys has done

**Comment by ArchiFleKs** [comment]
@saxenachaitanya hi and thanks. Everything is in this repository or the repository associated with the module (either the official one or the particule one)

This project is pure terragrunt / terrragrunt and some knowledge of internal working of Kubernetes as well as common addons. 

If you have more specific questions don't hesitate.
